app/get-your-ticket/index.html

```diff
@@ -3,12 +3,12 @@
 title: Get Your Ticket
 ---
 
-<h1>Every Kid in a Park - Get a Ticket</h1>
+<h1>Every Kid in a Park - Get a Paper Voucher</h1>
 
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+<p>On the next page you'll get your paper voucher and important instructions on how to redeem the voucher for the free annual pass.  In order to process your voucher, we first need the ZIP Code area for your school.  You can enter it in the box below.</p>
```

Comment:
Is it the zip code for the school? Or is it the zip code where they live? Did we decide? 

I suspect they might know the zip code where they live, but not the one for their school.